Create ui-c8y-1024-5-2-fix-c-8-y-select-dropdown-menu-scrollbar-12465-graft-release.md (#4942)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1024-5-2-fix-c-8-y-select-dropdown-menu-scrollbar-12465-graft-release.md

---
date: ""

> title: Fix c8y select dropdown menu scrollbar

Reviewer: ```suggestion
title: Fixed layout issues of select dropdown menu scrollbar
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67161
version: 1024.5.2
---

> Fixed the scrollbar not displaying correctly in select dropdown menus, making long lists of options hard to navigate.

Reviewer: ```suggestion
The scrollbar did not display correctly in select dropdown menus, making long lists of options hard to navigate. This issue has been fixed.
```

---
date: ""
title: fix c8y select dropdown menu scrollbar (#12465) [GRAFT][release/cd] (#12571)
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67161
version: 1024.5.2
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-67161] fix c8y select dropdown menu scrollbar
(#12465)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12465)

<!--- Backport version: unknown -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-67161]:
https://cumulocity.atlassian.net/browse/MTM-67161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
Fixed the scrollbar not displaying correctly in select dropdown menus, making long lists of options hard to navigate.
```
